vignettes/MSnbase-centroiding.Rmd

```diff
@@ -136,35 +146,38 @@ After centroiding the data consists of a single intensity for each
 mass peak. In the example above the centroids from consecutive scans
 do however not have the same m/z value, but they fluctuate between the
 discrete m/z values defined by the instruments's resolution. For lower
-intensity signals this variation can be substantial. To illustrate
-this we plot below the centroided signal for the [M+H]+ ion of
-proline.
+intensity signals this variation can be substantial.
+
+To further illustrate this, we plot below the centroided signal for
+the [M+H]+ ion of proline.
 
 ```{r proline, fig.align = "center", warning = FALSE, fig.cap = "Centroided data for proline."}
 prol_mz <- 116.070608
 prol_mzr <- c(prol_mz - 0.01, prol_mz + 0.01)
 prol_rtr <- c(165, 175)
 
 proline <- data_prof %>%
+	pickPeaks() %>%
```

Comment:
while this is more intuitive, filtering prior to `pickPeaks` can be beneficial for the performance. But basically, I'm fine with this.